### PR TITLE
Add support for build and use Strawberry Perl 5.24.4.1 (Apr 2018)...

### PR DESCRIPTION
**Pull request by tabad**
The files *`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-1.*`* have been uploaded to [SourceForge](https://sourceforge.net/projects/fiawi/files/strawberry-perl-packages/).

In addition, test files have been uploaded with FusionInventory-Agent v2.4.1-rc3:

  * [*`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-1-fia-2.4.1-rc3-tests-x64.txt`*](https://bit.ly/2luvyzk)
  * [*`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-1-fia-2.4.1-rc3-tests-x86.txt`*](https://bit.ly/2IpdShm)

The package  [*`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-1.tar.xz`*](https://bit.ly/2ts1CIz) seems to be correctly constructed but the tests are flawed.

Regards,

### CODE REVIEW

File: Perl/Scripts/load-perl-environment [hunk 1]
```diff
@@ -94,8 +94,8 @@ declare -r fusinv_agent_mod_name='fusioninventory-agent'
...
+declare -r fusinv_agent_mod_uses='base Config constant Crypt::DES Crypt::Rijndael Cwd Data::Dumper Digest::MD5 Encode Encode::Byte Encode::CN Encode::JP Encode::KR Encode::TW Encode::Unicode English Fcntl File::Basename File::Find File::Glob File::Path File::Spec File::stat File::Temp Getopt::Long HTTP::Cookies HTTP::Headers HTTP::Request HTTP::Status integer IO::Handle lib List::Util LWP Memoize MIME::Base64 Net::Domain Net::hostent Net::HTTPS Net::SNMP::Security::USM Net::SNMP::Transport::IPv4::TCP Net::SNMP::Transport::IPv6::TCP Net::SNMP::Transport::IPv6::UDP Pod::Usage POSIX Scalar::Util Socket Storable strict Sys::Hostname Sys::Syslog Thread::Semaphore threads threads::shared Tie::Hash::NamedCapture Time::HiRes Time::Local Time::localtime Time::Piece URI User::pwent utf8 warnings Win32 Win32::API Win32::OLE::Const Win32::OLE::Variant'
```
Comment (g-bougard) [inline Perl/Scripts/load-perl-environment]: List::Util & Time::Piece dependencies were removed in Agent, why add it again here ?
See 7c6ddd49959ce396ac9756bf238f09af573b0a63 & 8568217862f1140a4bd0085ecf24700798c0bb33

File: Perl/Scripts/load-gnu-utilities-environment.bat [hunk 2]
```diff
@@ -50,8 +50,12 @@ set HOME=%USERPROFILE%
 set USER=%USERNAME%
 set HISTFILE=%HOME%
 
-copy /y ..\..\Tools\7zip\x86\7za.exe "%TEMP%" > NUL
+copy /y ..\..\Tools\7zip\x86\7z.exe "%TEMP%" > NUL
+copy /y ..\..\Tools\7zip\x86\7z.dll "%TEMP%" > NUL
 copy /y ..\..\Tools\Curl\x86\curl.exe "%TEMP%" > NUL
+copy /y ..\..\Tools\Curl\x86\brotlidec.dll "%TEMP%" > NUL
```
Comment (g-bougard) [inline Perl/Scripts/load-gnu-utilities-environment.bat]: What are the brotli*.dll DLLs ?

**Comment by tabad** [comment]
Hello @g-bougard,

> You should better have submit your PR against https://github.com/TECLIB/fusioninventory-agent-windows-installer/ which is the repo I uses for rc releases.

I'm sorry, I didn't know what you preferred. Should I create again the PR inside https://github.com/TECLIB/fusioninventory-agent-windows-installer/?

Regards,

File: Perl/Scripts/load-perl-environment (hunk 1 above)
Comment (tabad) [reply Perl/Scripts/load-perl-environment]: Now I don't remember why. I'll take a look at it.

File: Perl/Scripts/load-gnu-utilities-environment.bat (hunk 2 above)
Comment (tabad) [reply Perl/Scripts/load-gnu-utilities-environment.bat]: These are regarding commit *[Update to cURL v7.60.0](49a2478 )*. Please, take a look at its comments. More info [here](https://en.wikipedia.org/wiki/Brotli).

File: Perl/Scripts/load-perl-environment (hunk 1 above)
Comment (tabad) [reply Perl/Scripts/load-perl-environment]: Yes, you are right. I'm going to fix it.

**Comment by tabad** [comment]
Uploaded files with tests of FusionInventory-Agent 2.4.1 with Strawberry Perl 5.24.4.1.

  * [`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-2-fia-2.4.1-tests-x64.txt`](https://bit.ly/2NgEvsM)
  * [`strawberry-perl-5.24.4.1-all-pepfia-2.4.x-2-fia-2.4.1-tests-x86.txt`](https://bit.ly/2KDUvmT)

**Pull request closed without merging**